src/apps/content-editor/src/app/components/Nav/ContentNav.less

```diff
@@ -72,7 +74,7 @@
 .NavWrap {
   white-space: normal;
   position: relative;
-  padding-bottom: 50px;
+  // padding-bottom: 50px;
```

Comment:
remove

Reply:
Inline styles is the way we are moving forward with the migration as discussed. This follows our Marketing site and MUI conventions.

CC: @agalin920

Reply:
please take a second look at this change request

Reply:
With 50px 
<img width="388" alt="Screen Shot 2022-05-24 at 10 34 46 AM" src="https://user-images.githubusercontent.com/22800749/170097546-d567013e-b68f-4b2b-b453-78a8a71ba3f3.png">

Without 50p
<img width="460" alt="Screen Shot 2022-05-24 at 10 35 38 AM" src="https://user-images.githubusercontent.com/22800749/170097731-38be4fa4-81d1-4c76-9f7a-42baee487243.png">
x

Reply:
sorry should have been clearer. please remove the commented out code. no need to leave that lying around

Reply:
also what changed to cause needing this style change? layout seemed to be fine before.

Reply:
@shrunyan Yes I have reverted this one and brought back the 50px it adds some more breathing room.

Reply:
https://github.com/zesty-io/manager-ui/pull/1053/files/ffb38ff3a8548575e10fe5b3a0fa6b595bd9ecc1#diff-88a95acfd1c51e6594b4a35ce63dbe7f05dc0b004be9745c6ccbdeadf53230efR77

still shows as commented out

Reply:
This is looking at an outdated commit. 
https://github.com/zesty-io/manager-ui/blob/content-mui-button-migration/src/apps/content-editor/src/app/components/Nav/ContentNav.less#L77